internal/build/docker.go

```diff
@@ -298,6 +302,9 @@ func readDockerOutput(ctx context.Context, reader io.Reader) (*json.RawMessage,
 		if len(message.Stream) > 0 && message.Stream != "\n" {
 			msg := strings.TrimSuffix(message.Stream, "\n")
 			output.Get(ctx).Print("%+v", msg)
+			if strings.HasPrefix(msg, "Step") || strings.HasPrefix(msg, "Running") {
+				innerSpan, ctx = opentracing.StartSpanFromContext(ctx, fmt.Sprintf("%s", msg))
```

Comment:
msg is already a string? you shouldn't need to sprintf it

Reply:
oh yeah this was cruft from before, thanks